### PR TITLE
Powerng 46

### PR DESCRIPTION
**Pull request by miksik98**

### CODE REVIEW

File: src/main/java/com/uxtest/backend/dto/UserDTO.java [hunk 1]
```diff
@@ -39,8 +39,6 @@
 
     private Boolean recordingAgreement;
 
-    private List<CardSortingResultDTO> results;
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/dto/UserDTO.java:42]: Is not 'result' a jsonproperty? Mark all of them with @JsonProperty or none.

File: src/main/java/com/uxtest/backend/model/cardsorting/CategoryWithSubjects.java [hunk 2]
```diff
@@ -33,14 +34,13 @@
     @JoinColumn(name = "cardSortingResult_id")
     private CardSortingResult result;
 
-    @OneToMany(mappedBy = "categoryWithSubjects")
-    private List<Subject> subjects;
+    @ManyToMany(mappedBy = "categoriesWithSubjects")
+    private List<Subject> subjects = new ArrayList<>();
```
Comment (michalbarczyk) [inline src/main/java/com/uxtest/backend/model/cardsorting/CategoryWithSubjects.java:38]: Should it really be instantiated here? As I don't know a  context is seems weird but maybe it is just ok

File: src/main/java/com/uxtest/backend/dto/UserDTO.java (hunk 1 above)
Comment (miksik98) [reply src/main/java/com/uxtest/backend/dto/UserDTO.java:42]: I have only deleted relation between User and CardSortingResult. It's not obligated now.

File: src/main/java/com/uxtest/backend/model/cardsorting/CategoryWithSubjects.java (hunk 2 above)
Comment (miksik98) [reply src/main/java/com/uxtest/backend/model/cardsorting/CategoryWithSubjects.java]: Thank you, I was debugging sth and forgot about deleting this intialization. Now it is deleted.

**Pull request merged**